NineChronicles.Standalone/NineChroniclesNodeService.cs

```diff
@@ -148,6 +156,7 @@ public IHostBuilder Configure(IHostBuilder hostBuilder)
                         services.AddHostedService(provider => new ActionEvaluationPublisher(
```

Comment:
We may need to rename this publisher... 😓

Reply:
Yes I agree. I'll make an issue for this :)